### ISSUE TITLE
The Script has stopped working for me.  

### ISSUE THREAD

**Issue by rteitelman** [magma1447/greasemonkey-geocaching-projectgc#158]
tried both Firefox and Chrome.  Not seeing anything working.  I see the script was just updated to version 2.4.11 on May 8 2024.  Could that be the issue?

**Comment by magma1447** [comment]
Latest version is `2.4.11` and is 6 months old.

Project-GC is having some hosting issues today, that might be the reason the script doesn't work for you. Most requests to Project-GC will fail. That should have been resolved now, but it isn't. But within 30 minutes it shall be resolved for real (hopefully).

**Comment by rteitelman** [comment]
Yes, it's working now again.   Funny how the Greasemonkey script on my
laptop says version 2.4.11 was last updated on May 8, 2024.  don't know why
it says that if the script is 6 months old.   But in any event, it's
working!  thanks.
